server/upstream/uphttp/upstream.go

```diff
@@ -57,7 +63,7 @@ func (u *Upstream) invoke(fromMattermostUserID string, call *apps.CallRequest) (
 
 // post does not close resp.Body, it's the caller's responsibility
 func (u *Upstream) post(fromMattermostUserID string, url string, msg interface{}) (*http.Response, error) {
-	client := u.getClient()
+	client := u.httpOut.MakeClient(true)
```

Comment:
To double check: This is fine because there is another layer of security (JWT) on top of the http connection, right? Would we still recommend customers to not use `localhost`?

Reply:
The app's RootURL is declared and accepted by the admin upon its install, so we can trust it.

Reply:
The `RootURL` is only accepted indirectly. What the admin adds is a _manifest_ pointing to an URL. Currently, it's not checked if the manifest is under the `RootURL`, right?

Reply:
Added HTTPRootURL to the consent screen. The manifest can come from anywhere, really, but the Root URL should be consented to.
![image](https://user-images.githubusercontent.com/1187448/117447231-14402980-aef2-11eb-87e4-54b9eb19a059.png)

Reply:
Where on the screenshot is the "Rooturl" permission you are referring to @levb ?

Reply:
@aaronrothschild Access Remote HTTP API...